src/pane-resize-handle-view.coffee

```diff
@@ -0,0 +1,67 @@
+{$, View} = require 'atom-space-pen-views'
```

Comment:
Can you remove all references of view and jQuery? We're no longer using space pen or jQuery in core. In place of jQuery, please use the raw DOM APIs.